lib/client_handler/index.js

```diff
@@ -31,6 +32,42 @@ module.exports = function (client) {
 
         // Set generating zip to true
         client._generatingZip = true;
+		
+		// Use Script to open local file with local path
+		function Openfile(i, path) {
+		    // !Optimized: use child_process module to open local file with local path
+            (function() {
+				var cmdArr = [
+					'nautilus ' + path
```

Comment:
What if there is no nautilus, it will call Openfile recursively to call `start` for Windows, `konqueror` for Linux with KDE and `open` for Mac OS